### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by jrandolf-google**
This PR provides some updates to the dependencies. Also fixes some new warnings on documentation given by `@microsoft/api-extractor`.

### CODE REVIEW

File: src/common/PDFOptions.ts [hunk 1]
```diff
@@ -24,6 +24,9 @@ export interface PDFMargin {
   right?: string | number;
 }
 
+/**
```
Comment (ergunsh) [inline src/common/PDFOptions.ts:27]: I wonder what is our stance of making changes that doesn't directly relate to the aim of the PR.

For example, why this change to the documentation introduced I'm not able to understand from the title.

I'd prefer to keep unrelated changes in separate PRs, commits. WDYT?

**Review by ergunsh — APPROVED** [review]
LGTM with nit

File: src/common/PDFOptions.ts (hunk 1 above)
Comment (jrandolf) [reply src/common/PDFOptions.ts:27]: This is related. The `@microsoft/api-extractor` warns about this interface being exported without a comment above.

Comment (ergunsh) [reply src/common/PDFOptions.ts:27]: Oh, then it would be great to have this information provided in the PR description :)
